frontends/precinct-scanner/src/screens/election_manager_screen.tsx

```diff
@@ -111,7 +111,11 @@ export function ElectionManagerScreen({
       openToggleLiveModeWarningModal();
     } else {
       setIsLoading(true);
+      const minimumDelay = new Promise((resolve) => {
+        setTimeout(resolve, 2000);
+      });
```

Comment:
Could have been this:
```suggestion
      const minimumDelay = sleep(2000);
```